Documentation/ApiOverview/Typo3CoreEngine/Database/Index.rst

```diff
@@ -293,6 +293,19 @@ Examples of Commands:
    $cmd['tt_content'][1203]['move'] = 400;  // Moves tt_content uid=1203 to the first position in page uid=400
 
 
+.. _tce-data:
+
+Accessing the uid of copied records:
+---------------------
```

Comment:
The underline for headers must cover the whole text, so until the end of the line.